src/middleware.ts

```diff
@@ -234,7 +234,7 @@ export const persist = <S extends State>(
 
   api.setState = (state, replace) => {
     savedSetState(state, replace)
-    setItem()
+    void setItem()
```

Comment:
This was added due to:

![Screen Shot 2021-03-26 at 8 44 12 AM](https://user-images.githubusercontent.com/11803153/112640750-a04d3480-8e0f-11eb-98cc-675284cbb571.png)